chromium_src/ios/chrome/browser/policy/model/configuration_policy_handler_list_factory.mm

```diff
@@ -0,0 +1,30 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "ios/chrome/browser/policy/model/configuration_policy_handler_list_factory.h"
+
+#include <memory>
+
+#include "brave/ios/browser/policy/brave_simple_policy_map_ios.h"
+#include "components/policy/core/browser/configuration_policy_handler.h"
+
+#define BuildPolicyHandlerList BuildPolicyHandlerList_ChromiumImpl
+#include <ios/chrome/browser/policy/model/configuration_policy_handler_list_factory.mm>
+#undef BuildPolicyHandlerList
+
+std::unique_ptr<policy::ConfigurationPolicyHandlerList> BuildPolicyHandlerList(
+    bool are_future_policies_allowed_by_default,
+    const policy::Schema& chrome_schema) {
+  std::unique_ptr<policy::ConfigurationPolicyHandlerList> handlers =
+      BuildPolicyHandlerList_ChromiumImpl(
+          are_future_policies_allowed_by_default, chrome_schema);
+
+  for (const auto& entry : policy::kBraveSimplePolicyMap) {
```

Comment:
so per recent post in browser-dev rather than add implementations here (with the associated addition deps) we should just have a declaration and put the impl in a separate target that is a direct dep of brave/ios/browser. This avoids the need for the deps patch